robottelo.properties.sample

```diff
@@ -79,3 +79,44 @@ driver=firefox
 # Default set to be 0, i.e. no timing of performance is measured and thus no
 # interference to original robottelo tests.
 test.foreman.perf=0
+
+# Folowing entries are used for preparation of performance tests after a fresh
+# install. They will be used by
+# `test/foreman/performance/test_standard_prep.py`, which supports:
+#
+# 1. downloading manifest,
+# 2. uploading manifest to subscription,
+# 3. updating Red Hat CDN URL,
+# 4. enabling key repositories: rhel6-rpms, rhel7-rpms, rhel6-kickstart-rpms,
+#    rhel7-kickstart-rpms, rhel6-optional-rpms, rhel7-optional-rpms,
+#    rhel6-optional-source-rpms, rhel7-optional-source-rpms,
+#    rhel6-optional-debug-rpms, r7-optional-debug-rpms
+#
+# Note that this preparation step is not required as long as satellite server
+# is already configured.
+test.cdn.address=http://example.redhat.cdn/pub
+
+# Set variables for a list of virtual machines' IP address, which would conduct
+# concurrent system subscriptions. Assume that those vms are pre-provisioned.
+# As for current workaround, append each IP address to the end, join by ","
+# without any spaces.
+test.virtual_machines_list=127.0.0.1,127.0.0.1,127.0.0.1,127.0.0.1,127.0.0.1
+
+# Savepoint utility to restore the database. For example, after conducting
+# 5,000 concurrent subscription by activation-key using 10 clients, in order to
+# start next 5k test case of subscription by register and attach, the
+# performance test would restore the database back to the state where there's
+# no client being registered.
+#
+# Create savepoint-1 immediately after a fresh installation of Satellite.
+test.savepoint1_fresh_install=date-savepoint1-description
+#
+# Create savepoint-2 after enabling repositories, but before any system
+# subscription or repository synchronization.
+test.savepoint2_enabled_repos=date-savepoint2-repos-enabled-not-sync
+
+# Parameter for number of threads for each test case
+csv.num_threads=1,2,4,6,8,10
```

Comment:
The effect of this setting is unclear. What test case will use this setting? I can use `git grep`, but it would be much more convenient if you this docstring states something like:

> Class `foo` and its subclasses use this setting when doing X.